ChangeLog

```diff
@@ -1,3 +1,13 @@
+2024-13-10  Contantinos Giachalis  <cgiachalis@outlook.com>
+
+  * src/interface.cpp (log_setup): Documentation fix for duplicate log level
+  
+  * man/log_setup.Rd: Idem
+  
+  * R/RcppExports.R: Idem
+	
+  * DESCRIPTION (RoxygenNote): Idem
```

Comment:
Baby nag:  No empty lines between items that belong together, and I usually do mention DESCRIPTION for the roxygen note (it will fall back to 6.* from processing here anyway).  I can deal with though.

Oh, and an eight space tab before the *.  I can clean this up.  Thanks for adding it!

Reply:
> No empty lines between items that belong together
Duh.. Changed it just before I push ..